pipelines/README.md

```diff
@@ -117,6 +117,12 @@ Create a copy of the file(s) below to include the required credentials for acces
     # Linking secret is needed only if the secret from the Robot account hasn't been already applied and linked in the previous step
     $ oc secret link pipeline credentials-image-registry
     ```
+### Setup ODH Model Registry for Storing Models
```

Comment:
The one in this PR is the link to the MR operator repo which is changing fast. 
@devguyio Maybe we would need to update the CLI with the link for the MR installations because anyone using the latest changes of MR operators would see errors?

Reply:
Unless there is some custom configuration for model registry, I agree that we should defer to the instruction in the model-registry-operator repo.  If we notice any issues with failed deployments then it would be best for the project to file bugs / PRs to improve their instructions then us owning a duplicate set of instructions.

Reply:
So, leaving this as is. We can open a PR to the CLI readme if needed to reflect the same change.